### PR TITLE
Add chainerx tests to test_create_mnbn_model

### PR DESCRIPTION
**Pull request by keisukefukuda**
This PR is a part of #8031. It adds ChainerX tests to test_create_mnbn_model.py

### CODE REVIEW

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py [hunk 1]
```diff
@@ -47,14 +51,16 @@ def check_create_mnbn_model_chain(self, gpu):
         self.assertTrue(
             isinstance(mnbn_model.bn,
                        chainermn.links.MultiNodeBatchNormalization))
-        if gpu:
-            mnbn_model.to_device(cupy.cuda.Device())
+        device = get_device(self.communicator.intra_rank if use_gpu else None,
+                            use_chx)
+        device.use()
```
Comment (belldandyxtq) [inline tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: Unnecessary

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py [hunk 2]
```diff
@@ -63,13 +69,16 @@ def check_create_mnbn_model_chain_list(self, gpu):
         self.assertTrue(
             isinstance(mnbn_model[1],
                        chainermn.links.MultiNodeBatchNormalization))
-        if gpu:
-            mnbn_model.to_device(cupy.cuda.Device())
+        device = get_device(self.communicator.intra_rank if use_gpu else None,
+                            use_chx)
+        device.use()
```
Comment (belldandyxtq) [inline tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: Unnecessary

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py [hunk 3]
```diff
@@ -80,29 +89,38 @@ def check_create_mnbn_model_sequential(self, gpu):
         mnbn_model = chainermn.links.create_mnbn_model(model,
                                                        self.communicator)
 
-        if gpu:
-            mnbn_model.to_device(cupy.cuda.Device())
+        device = get_device(self.communicator.intra_rank if use_gpu else None,
+                            use_chx)
+        device.use()
```
Comment (belldandyxtq) [inline tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: Unnecessary

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py (hunk 1 above)
Comment (keisukefukuda) [reply tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: fixed.

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py (hunk 3 above)
Comment (keisukefukuda) [reply tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: fixed.

File: tests/chainermn_tests/links_tests/test_create_mnbn_model.py (hunk 2 above)
Comment (keisukefukuda) [reply tests/chainermn_tests/links_tests/test_create_mnbn_model.py]: fixed.

**Comment by belldandyxtq** [comment]
pfnci, test this please

**Pull request merged**